scripts/facts.runs/format_facts.py

```diff
@@ -78,5 +76,4 @@
 save = Path(os.getcwd())
 save = save.parent.absolute() / 'input' / 'climate'
 
-gmsl_ds.to_netcdf(save / gmsl_pulse, encoding = {"control_gmsl":{"dtype":"float64"},"pulse_gmsl":{"dtype":"float64"}})
-    
\ No newline at end of file
+gmsl_ds.to_netcdf(gmsl_pulse, encoding = {"control_gmsl":{"dtype":"float64"},"pulse_gmsl":{"dtype":"float64"}})
```

Comment:
@JMGilbert can you add a comment on line 74 (`gmsl_ds = xr.merge([control,pulse])/10`) to say what it's for (I'm assuming it's a unit conversion)